### ISSUE TITLE
Intergrate TravisCI

### ISSUE THREAD

**Issue by SillySam** [SillySam/Sync-Tunnel#5]
Create unit tests and integrate them into Travis CI, also add the pass/fail button to README.

**Comment by Sukhj1nder** [comment]
Are you going to setup Travis CI for this repository?
As, I think that only the owners of the repo can integrate Travis into the project.

https://matthewmoisen.com/blog/how-to-set-up-travis-ci-with-github-for-a-python-project/

**Comment by SillySam** [comment]
Yes and No. I want unit tests made as well as a TravisCI config which I will then add into the project.
